recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "4.2" %}
```

Comment:
I really don't like this practice of intentionally backdating recipes to be accepted.  How is this any better for you or anyone else than adding the current version, and you adding the old version in the feedstock?